utensor_cgen/operators.py

```diff
@@ -0,0 +1,125 @@
+from .snippets import *  # pylint: disable=W0401
+from .snippets import register_template
+from ._snippets_base import SnippetContainer, Snippet
+
+
+class _Operator():
+  # Base Class
+  snippet = None
+  name = ""
+
+  def getSnippet(self):
+    return self.snippet
```

Comment:
Are you trying to make `self.snippet` read only so you write a getter here?
Why not use `@property`?

Reply:
not to familiar with `@property`, XD If it works better than we can change it :)

Reply:
yeah, `property` seems like the way to go.